core/src/avm2/globals/flash/events/AudioOutputChangeEvent.as

```diff
@@ -4,6 +4,7 @@
 // It won't be regenerated in the future, so feel free to edit and/or fix
 package flash.events
 {
+    // TODO: [API("724")]
```

Comment:
I think mentioning Flash Player 27, and that `724` is just a guess/extrapolation, would be better.